BedrockServer.cpp

```diff
@@ -9,7 +9,20 @@ set<string>BedrockServer::_blacklistedParallelCommands;
 recursive_mutex BedrockServer::_blacklistedParallelCommandMutex;
 
 void BedrockServer::acceptCommand(SQLiteCommand&& command) {
-    _commandQueue.push(BedrockCommand(move(command)));
+    // If the sync node tells us that a command causes a crash, we immediately save that.
+    if(SIEquals(command.request.methodLine, "CRASH_COMMAND")) {
```

Comment:
Can you update this code to reduce the scope of the `unique_lock` to not include the syslog call?  I wager it can be pretty slow.  So how about something like:

```
SData request;
request.deserialize(command.request.content);
{
    //  Atomically update the blacklisted crash command
    unique_lock<decltype(_crashCommandListMutex)> lock(_crashCommandListMutex);
    _crashCommands.insert(make_pair(request.methodLine, request.nameValueMap));
}
SALERT("Blacklisting command (now have " << _crashCommands.size() << " blacklisted commands): " << request.serialize());
```